cmd/manager/exec/manager.go

```diff
@@ -65,6 +65,7 @@ func RunManager() {
 	if Options.Standalone {
 		// for standalone subcription pod
 		leaderElectionID = "multicloud-operators-standalone-subscription-leader.open-cluster-management.io"
+		metricsPort = 8389
 	} else if !strings.EqualFold(Options.ClusterName, "") && !strings.EqualFold(Options.ClusterNamespace, "") {
 		// for managed cluster pod appmgr. It could run on hub if hub is self-managed cluster
 		leaderElectionID = "multicloud-operators-remote-subscription-leader.open-cluster-management.io"
```

Comment:
Good catch, Let's add different metricPort in remote cluster mode as well.  As we discussed,  when hub becomes self-managed cluster, the subscription controllers of the three modes could run on the hub.

Reply:
adding it